### PR TITLE
 Add stage for generation of Resource Usage Report with Plot plugin

### PR DESCRIPTION
**Pull request by Nancy-Chauhan**
- [x] Fixes the issue of Performance Plugin for the Yosys Reports https://ci.librecores.org/job/Projects/job/OpenRISC/job/mor1kx/job/master/16/consoleFull

- [x] [Plot Plugin](https://wiki.jenkins.io/display/JENKINS/Plot+Plugin) is used to generate graphical reports, which only uses CSV format of key Value pairs

cellcount

CC @oleg-nenashev @stffrdhrn

### CODE REVIEW

**Comment by stffrdhrn** [comment]
This looks good. Do you plan to move this to the pipeline lib? It seems it would cause conflicts with the other pull request.

**Comment by Nancy-Chauhan** [comment]
@stffrdhrn , That work is still in progress, So We can keep yosys keep going in the existing pipeline

File: Jenkinsfile [hunk 1]
```diff
@@ -9,26 +9,52 @@ pipeline {
...
             }
         }
 
-        stage('Resource Usage Report Generation') {
+        stage('Generate Resource Usage Report') {
             steps {
-                echo "-=- execute performance tests -=-"
-                perfReport 'output.csv'
+                echo "-=- report metrics -=-"
+                plot csvFileName: 'plot-yosys-stats.csv', csvSeries: [
```
Comment (oleg-nenashev) [inline Jenkinsfile]: Looks like the code below belongs to the Pipeline library

**Comment by stffrdhrn** [comment]
@Nancy-Chauhan do we need this PR still?

**Comment by Nancy-Chauhan** [comment]
No @stffrdhrn, We don't need this PR. Closing it with development of the PR https://github.com/openrisc/mor1kx/pull/93

**Pull request closed without merging**